configure

```diff
@@ -211,6 +211,7 @@ touch s.h m.h Makefile
 # Write options to Makefile
 
 echo "# generated by ./configure $configure_options" >> Makefile
+echo "CONFIGURE_ARGS=$configure_options" >> Makefile
```

Comment:
This will fail as soon as there are spaces in some `configure` argument, which is pretty common under Windows.